site/_scss/blocks/_post-banner.scss

```diff
@@ -0,0 +1,23 @@
+.post-banner {
+  background-color: #fff;
+  border-radius: 6px;
+  margin-bottom: 30px;
+  position: sticky;
+  top: 24px;
+  z-index: 9;
+
+  .post-banner__content {
```

Comment:
This would produce a selector `.post-banner .post-banner__content`, which is both too specific and too long. Use `&__content` instead which produces the shorter `.post-banner__content` selector.